### PR TITLE
feat: creating home page

### PR DESCRIPTION
**Pull request by mariyasada**
creating the home page and product list of E-commerce.  do review it

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -11,13 +12,15 @@ body{
     margin:0;
     padding: 0;
```
Comment (SaurabhChawda) [inline main.css:13]: You can add box-sizing: border-box here.

**Review by SaurabhChawda — APPROVED** [review]

File: stylesheets/product-list/productList.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+.price
+{
+    margin-left: -2rem;
+    margin-right: 1rem;
+}
+/* .rating-with-icon
+{
+   max-width:3.5rem;
+   flex-shrink: 2;
+} */
```
Comment (harshitadaswani) [inline stylesheets/product-list/productList.css]: remove comments

**Review by harshitadaswani — APPROVED** [review]
remove the comments and you are good to go

File: main.css (hunk 1 above)
Comment (mariyasada) [reply main.css:13]: I have added this property to the body element

File: stylesheets/product-list/productList.css (hunk 2 above)
Comment (mariyasada) [reply stylesheets/product-list/productList.css]: okay.

**Pull request merged**